TestVectors/runtimes/rust/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "aws-esdk-test-vectors"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.80.0"
+keywords = ["test-vectors", "testvectors", "test vectors", "esdk test vectors", "esdk dafny test vectors", "cryptography", "security", "encryption", "client-side"]
```

Comment:
Are we publishing this as a separate crate?

Reply:
We don't need to publish this, I just added these for completeness but I can remove the publishing stuff.